website/client/components/chat/chatCard.vue

```diff
@@ -221,6 +223,11 @@ export default {
       }
       return this.icons[`tier${message.contributor.level}`];
     },
+    tierTitle () {
+      const message = this.msg;
+      const isContributor = Boolean(message.contributor && message.contributor.level);
```

Comment:
I don't think this check is necessary as the ´contribText´ function makes the same exact change so in case the user is not a contributor it'll just return undefined and at that point you can return an empty string: `return achievementsLib .getContribText(...) || ''`